docs/pages/components/DropdownMenu.md

```diff
@@ -0,0 +1,20 @@
+# DropdownMenu
+
+Primitive DropdownMenu component with variants
+
+```.jsx
+  <DropdownMenu
+    options={[
+      { label: 'Menu Item One', value: 'A0' },
+      { label: 'Menu Item Two', value: 'B0' },
+      { label: 'Menu Item Three', value: 'C0' },
+      { label: 'Menu Item Four', value: 'D0' },
+      { label: 'Menu Item Five', value: 'E0' },
+      { label: 'Menu Item Six', value: 'F0' },
+      { label: 'Menu Item Seven', value: 'G0' },
+    ]} 
+    onSelect={(value, event) => console.log(value)}
+  >
+    Trigger &#9662;
+  </DropdownMenu>
+```
```

Comment:
- [ ] This is not the right way to format editable code. Please check other components doc.

Reply:
do you mean trigger text?

Reply:
No. I meant, the editable code. The code formatting should be something like this
<img width="342" alt="Screenshot 2020-04-01 at 8 23 42 AM" src="https://user-images.githubusercontent.com/5488330/78094433-29df4780-73f2-11ea-952f-9c4f1cfcb2e5.png">